packages/video_player/android/src/main/java/io/flutter/plugins/videoplayer/VideoPlayerPlugin.java

```diff
@@ -337,6 +320,9 @@ private void onMethodCall(MethodCall call, Result result, long textureId, VideoP
         videoPlayers.remove(textureId);
         result.success(null);
         break;
+      case "orientation":
```

Comment:
The Dart side does not call this method?

Reply:
oops, that was a leftover. Will remove it.